app/views/hyrax/dashboard/collections/new.html.erb

```diff
@@ -1,10 +1,14 @@
-<% provide :page_title, construct_page_title( t('.header', type_title: @collection.collection_type.title) ) %>
+<% provide :page_title, construct_page_title( t('.header', type_title:  @collection.collection_type.title) ) %>
 <% provide :page_header do %>
-  <h1><span class="fa fa-edit"></span> <%= t('.header', type_title: @collection.collection_type.title) %> </h1>
+  <% a_title = t(:'hyrax.dashboard.collections.new.header') + " " + @collection.collection_type.title %>
```

Comment:
Why are you putting this in a string? You can use erb:
```
<%= t(:'hyrax.dashboard.collections.new.header') %> <%= @collection.collection_type.title %>
```

Reply:
Thanks @jcoyne! @elrayle - Do you want me to revert this change too.